README.md

```diff
@@ -73,19 +95,19 @@ Quick Start Guide
    }
    ```
 
-7. Make your application Nitro VM-capable by running container converter with the file from previous step
+7. Make your application Nitro VM-capable by running container converter with the file from previous step.
+   The converter by default pulls the input image and pushes the output image to remote repositories. These images are then cleaned up from the local docker cache. In our example, the output image push is disabled in the request json and to preserve the images in the docker cache, 'PRESERVE_IMAGES' environment variable is specified.
    ```bash
-      # Run from the root of the repository
-      cd tools/container-converter/target/debug
-      ./container-converter --request-file <path to file from step 4>
+      docker run --rm --name converter --user 0 --privileged -v /var/run/docker.sock:/var/run/docker.sock -e PRESERVE_IMAGES=input,result -v /tmp/req-files:/app converter --request-file /app/req.json
```

Comment:
Minor: Having the parameters in separate lines looks better and increases readability 

 docker run --rm 
    --name converter 
    --user 0 --privileged 
    -v /var/run/docker.sock:/var/run/docker.sock 
    -e PRESERVE_IMAGES=input,result 
    -v /tmp/req-files:/app converter 
    --request-file /app/req.json

Reply:
Since this is part of the README and inside the bash brackets, it will be available in a scrollable dialog box with a copy button. I think it is usable enough for now.